test/BUILD.gn

```diff
@@ -505,6 +505,7 @@ if (!is_android) {
       "//brave/browser/brave_stats/brave_stats_updater_browsertest.cc",
       "//brave/browser/browsing_data/brave_clear_browsing_data_browsertest.cc",
       "//brave/browser/devtools/brave_devtools_ui_bindings_browsertest.cc",
+      "//brave/browser/ephemeral_storage/ephemeral_storage_browsertest.cc",
```

Comment:
almost missed this. Adding new files to the main test build target should be avoided. Please put this file in it's own source set in brave/browser/ephemeral_storage/BUILD.gn and add a dep here

Reply:
and make sure to include all necessary deps from the headers here https://github.com/brave/brave-core/pull/6560/files#diff-ad1b5bc543791a83f0e762f32a01e88544b826f9ecdad47ce2b8a49461cf7a7bR8

Reply:
Okay. I think this should be fixed now.